.ci/scripts/prepare-spec-changes.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -uexo pipefail
+
+readonly REPO_NAME=${1}
+readonly SPECS_FILEPATH=${2}
+readonly REPO_DIR=".ci/${REPO_NAME}"
+
+git clone "https://github.com/elastic/${REPO_NAME}" "${REPO_DIR}"
+
+SPECS_DIR=$(dirname "${SPECS_FILEPATH}")
+mkdir -p "${REPO_DIR}/${SPECS_DIR}"
+
+echo "Copying files to the ${REPO_NAME} repo"
+cp -f spec/spec.json "${REPO_DIR}/${SPECS_FILEPATH}"
+
+cd "${REPO_DIR}"
+git config user.email
+git checkout -b "update-spec-files-$(date "+%Y%m%d%H%M%S")"
+git add "${SPECS_FILEPATH}"
+git commit -m "test: synchronizing specs"
```

Comment:
Is this a commit message that is wanted?

Reply:
This is the commit in the PR, therefore the person who squash and merge could potentially change the title of the PR. Since there was no specifications at the time of writing this, but using what it was already implemented for other similar projects.

I guess, this is something that can be potentially changed in the future with the message that the apm agents have agreed, do you think for the time being we can move forward?

Reply:
> do you think for the time being we can move forward?

Yup, definitely. Thanks.